<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Points2One</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Input vector layer</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QgsMapLayerComboBox" name="wInputLayer">
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QRadioButton" name="wCreatePolygons">
       <property name="text">
        <string>Create polygons</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="wCreateLines">
       <property name="text">
        <string>Create lines</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="wCloseLines">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Closed</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="wGroup">
     <property name="text">
      <string>Group features by</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QgsFieldComboBox" name="wGroupField">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="wSort1">
     <property name="text">
      <string>Sort vertices by</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QgsFieldComboBox" name="wSortField1">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="wSort2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Then by</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QgsFieldComboBox" name="wSortField2">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Output shapefile</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="wOutputFileName">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="wBrowse">
       <property name="text">
        <string>Browse</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Encoding</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="wEncoding">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="wAddResult">
       <property name="text">
        <string>Add result to canvas</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QProgressBar" name="wProgressBar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsFieldComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsfieldcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgis.gui</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>wInputLayer</tabstop>
  <tabstop>wCreatePolygons</tabstop>
  <tabstop>wCreateLines</tabstop>
  <tabstop>wCloseLines</tabstop>
  <tabstop>wGroup</tabstop>
  <tabstop>wGroupField</tabstop>
  <tabstop>wSort1</tabstop>
  <tabstop>wSortField1</tabstop>
  <tabstop>wSort2</tabstop>
  <tabstop>wSortField2</tabstop>
  <tabstop>wOutputFileName</tabstop>
  <tabstop>wBrowse</tabstop>
  <tabstop>wEncoding</tabstop>
  <tabstop>wAddResult</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>wCreateLines</sender>
   <signal>toggled(bool)</signal>
   <receiver>wCloseLines</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wSort1</sender>
   <signal>toggled(bool)</signal>
   <receiver>wSort2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wSort2</sender>
   <signal>toggled(bool)</signal>
   <receiver>wSortField2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>443</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>443</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wGroup</sender>
   <signal>toggled(bool)</signal>
   <receiver>wGroupField</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wInputLayer</sender>
   <signal>layerChanged(QgsMapLayer*)</signal>
   <receiver>wGroupField</receiver>
   <slot>setLayer(QgsMapLayer*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wInputLayer</sender>
   <signal>layerChanged(QgsMapLayer*)</signal>
   <receiver>wSortField1</receiver>
   <slot>setLayer(QgsMapLayer*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wInputLayer</sender>
   <signal>layerChanged(QgsMapLayer*)</signal>
   <receiver>wSortField2</receiver>
   <slot>setLayer(QgsMapLayer*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wSort1</sender>
   <signal>toggled(bool)</signal>
   <receiver>wSortField1</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>223</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
